docs/src/explanations/colors.md

```diff
@@ -117,8 +109,67 @@ text!(ax, 4, 0, text = ":viridis",
 ylims!(ax, -1, 2)
 f
 ```
-\end{examplefigure}
 
 The following is a list of all the colormaps accessible via a `Symbol` in Makie which are defined in ColorSchemes.jl:
 
-{{colorschemes}}
+### misc
+
+These colorschemes are not defined or provide different colors in ColorSchemes.jl
+They are kept for compatibility with the old behaviour of Makie, before v0.10.
+
+```@example colors
+using Markdown # hide
+using Makie.PlotUtils # hide
+Core.include(@__MODULE__, "colormap_generation.jl") # hide
```

Comment:
Couldn't this be regular `include`?

Reply:
threw a UndefVarError, if Documenter uses some kind of bare modules, they wouldn't have an include defined